docs/rules/a11y-no-visually-hidden-interactive-element.md

```diff
@@ -0,0 +1,55 @@
+# Ensures that interactive elements are not visually hidden (`github/a11y-no-visually-hidden-interactive-element`)
+
+<!-- end auto-generated rule header -->
+
+## Rule Details
+
+This rule guards against visually hiding interactive elements. If a sighted keyboard user navigates to an interactive element that is visually hidden they might become confused and assume that keyboard focus has been lost.
```

Comment:
Can we note the default class/type, as well as which interactive elements we're checking against?

I like the note you added in the code about why we're excluding `input`. Could we add it here too?